clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/ops/servergroup/StackChecker.groovy

```diff
@@ -0,0 +1,40 @@
+package com.netflix.spinnaker.clouddriver.openstack.deploy.ops.servergroup
```

Comment:
Missing a license header.

Also, new code should ideally be written in Java

Reply:
Oh? I missed the java preference. When did that happen?

Reply:
This one might be weird. The rest of the clouddriver-openstack module is written in groovy. Is switching this one file over to Java what we want to do?

Reply:
Right at the top of the [Contributing doc](https://www.spinnaker.io/community/contributing/back-end-code/#java), FWIW. Wonder if it's been that way all along and we just missed it.

I'm _hoping_ we can let this one slide and promise to java in the future.

Reply:
I also missed that other sub modules are mixing groovy/java source. If we're just talking about making it a java file inside the groovy source directory (as I saw in other sub modules) I have no problem with this.